apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -204,8 +195,17 @@ function WpcomBlockEditorNavSidebar() {
 			<div
 				aria-label={ __( 'Block editor sidebar', 'full-site-editing' ) }
 				// Waiting for jsx-a11y version bump to support aria-description attribute
-				// eslint-disable-next-line jsx-a11y/aria-props
-				aria-description={ dialogDescription }
+				// aria-description={
```

Comment:
I'm not sure why I cannot disable this rule, `jsx-a11y/role-supports-aria-props`

Reply:
YES!